src/main/java/org/folio/service/mapping/MappingServiceImpl.java

```diff
@@ -105,15 +105,10 @@ public MappingServiceImpl() {
 
   private List<Rule> getRules(MappingProfile mappingProfile, OkapiConnectionParams params) {
     List<Rule> rulesFromConfig = configurationsClient.getRulesFromConfiguration(params);
-    return CollectionUtils.isEmpty(rulesFromConfig) ? ruleFactory.create(mappingProfile) : appendRulesFromProfile(rulesFromConfig, mappingProfile);
-  }
-
-  private List<Rule> appendRulesFromProfile(List<Rule> rulesFromConfig, MappingProfile mappingProfile) {
-    if (mappingProfile != null && isNotEmpty(mappingProfile.getTransformations())) {
+    if (mappingProfile != null) {
```

Comment:
so this if is just for logger?

Reply:
Yes, it's just for logger, forgot to add check that rulesFromConfig not empty